Makefile

```diff
@@ -20,6 +21,8 @@ install:
 	install -m0755 $(PROG).bash "$(DESTDIR)$(SYSTEM_EXTENSION_DIR)/$(PROG).bash"
 	install -d "$(DESTDIR)$(BASHCOMPDIR)/"
 	install -m 644 pass-otp.bash.completion  "$(DESTDIR)$(BASHCOMPDIR)/pass-otp"
+	install -d "$(DESTDIR)$(FISHCOMPDIR)/"
+	install -m 644 pass.fish  "$(DESTDIR)$(FISHCOMPDIR)/pass.fish"
```

Comment:
I think it should be called `pass-otp` since it might interfere with `pass`?

Reply:
I actually tried that first, but it seemed like fish uses the filename to apply its completions to a specific command (in this case `pass`). If you know of a way to name this file `pass-otp.fish` I'm all for it

Reply:
But won't that overwrite the default `pass.fish`?

Reply:
Yes, that's why by default this `pass.fish` installs into `/etc/fish/completions` and reads the default `pass.fish` from `/usr/share/fish/vendor_completions.d/pass.fish`. That is where the default ones are on my system, but I can't guarantee that on other systems. That's why checkboxes 3 and 4 are still open

Reply:
Yeah, the setup for completions is kind of messy and I haven't looked at this for a while. Lemme check upstream and get back to you.

Reply:
So, the ideal solution would be for pass to delegate to extension completion scripts itself, similar to what it does for `pass git` and `pass grep` here: https://git.zx2c4.com/password-store/commit/?id=06f499994071bb6131244218b25d637103afe1d5. This should be a reasonable change to submit upstream.